modules/installation-mirror-repository.adoc

```diff
@@ -0,0 +1,101 @@
+* You configured a mirror registry to use in your disconnected environment and
+can access the certificate and credentials that you configured.
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page and modified it to include authentication to your mirror repository.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the
+link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Quay.io]
+repository to determine the version of {product-title} that you want to install.
+
+. Set the required environment variables:
++
+----
+$ export OCP_RELEASE=<release_version> <1>
+$ export AIRGAP_REG='<bastion_host_name>:5000' <2>
+$ export AIRGAP_REPO='<repo_name>' <3>
+$ export PRODUCT_REPO='openshift-release-dev' <4>
+$ export AIRGAP_SECRET_JSON='<path_to_pull_secret>${HOME}/pull-secret-2.json' <5>
+$ export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=${AIRGAP_REG}/${AIRGAP_REPO}:${OCP_RELEASE} <6>
+$ export RELEASE_NAME="ocp-release" <7>
+----
+<1> For `<release_version>`, specify the version number of {product-title} to
+install, such as `4.2.0`.
+<2> For `<bastion_host_name>`, specify the registry domain name for your mirror
+repository.
+<3> For `<repo_name>`, specify the name of the repository to create in your
+registry. Use a descriptive name, such as `ocp4/openshift4`.
+<4> The repository to mirror. For a production installation, you must specify
+`openshift-release-dev`.
+<5> For `<path_to_pull_secret>`, specify the absolute path to and file name of
+the pull secret that contains both your Red Hat and registry credentials, such as
+`${HOME}/pull-secret-2.json`.
+<6> The installation program uses the `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE`
+parameter value to identify your local mirrored repository. You must use the
+value that is shown.
+<7> The release mirror. For a production installation, you must specify
+`ocp-release`.
+
+. Mirror the repository:
++
+----
+$ oc adm -a ${AIRGAP_SECRET_JSON} release mirror \
+     --from=quay.io/${UPSTREAM_REPO}/${RELEASE_NAME}:${OCP_RELEASE} \
+     --to=${AIRGAP_REG}/${AIRGAP_REPO} \
+     --to-release-image=${AIRGAP_REG}/${AIRGAP_REPO}:${OCP_RELEASE}
+----
++
+This command pulls the release information as a digest. The command output
+resembles the following text:
++
+----
+Success
+Update image:  <bastion_host_name>:5000/ocp/release:<release_version>
+Mirror prefix: <bastion_host_name>:5000/ocp/release
+
+To use the new mirrored repository to install, add the following section to the install-config.yaml:
+
+imageContentSources:
+- mirrors:
+  - <bastion_host_name>:5000/<repo_name>/release
+  source: quay.io/openshift-release-dev/ocp-release
+- mirrors:
+  - <bastion_host_name>:5000/<repo_name>/release
+  source: registry.svc.ci.openshift.org/ocp/release
+
+
+To use the new mirrored repository for upgrades, use the following to create an ImageContentSourcePolicy:
```

Comment:
This part isn't useful for install, so I'd elide it.  We just need the output through the `imageContentSources` block above.